fdbserver/workloads/BackupAndParallelRestoreCorrectness.actor.cpp

```diff
@@ -219,29 +219,25 @@ struct BackupAndParallelRestoreCorrectnessWorkload : TestWorkload {
 	ACTOR static Future<Void> checkDB(Database cx, std::string when,
 	                                  BackupAndParallelRestoreCorrectnessWorkload* self) {
 
-		return Void();
```

Comment:
The checkDB is supposed to compare the kv pairs in DB before and after restore.
This info. can be useful when restore result is inconsistent with the expected result.
Because I haven't reached to the point of revising this function, I simply commented out this function to avoid too much printf, which will be removed eventually.

In this PR, I comment out the `compareDBKV` to avoid the printfs.